modules/ROOT/pages/_partials/cookbook-dw/map-an-object-key/out.json

```
@@ -0,0 +1,12 @@
+{
+  "(sheet)1": [
+
+  ],
+  "(sheet)2": [
+    {
+      "Id": "2",
+      "Access": "4444",
+      "Subteam": "1"
+    }
+  ]
+}
```

@kevintroller I run the script locally and that's the correct output, does it makes sense to you?
Did you run it on the playground? The wrong output seems to be because a bug from the playground
```suggestion
{
  "(sheet)1": [
    {
      "Id": 1.0,
      "Team": "Mule",
      "timestamp": "2019-09-26T16:37:54",
      "Access": 12431
    }
  ],
  "(sheet)2": [
    {
      "Id": 2.0,
      "Team": "Max",
      "timestamp": "2019-09-26T16:37:54",
      "Access": 81243
    },
    {
      "Id": 2.0,
      "Team": "Max Mule",
      "timestamp": "2019-09-26T18:00:54",
      "Access": 67676
    },
    {
      "Id": "2",
      "Access": "4444",
      "Subteam": "1"
    }
  ]
}
```

---

Thanks!, Yes, I was running the script in the playground, I was not aware of the bug. I'll commit your suggestion.